visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -37,14 +38,26 @@ export class CustomConfigHelper {
 
 	static loadCustomConfigs() {
```

Comment:
[Optional]
rename to `loadCustomConfigsFromLocalStorage` to better distinguish from `importCustomConfigs`